file_formats/fip.md

```diff
@@ -20,46 +20,72 @@ For example:
 
 ```plaintext
 📦 fib
-┣ FIP_DataG_2024-06-05T08_25_33.csv
-┣ FIP_DataIso_2024-06-05T08_25_33.csv
-┣ FIP_DataR_2024-06-05T08_25_33.csv
-┣ FIP_RawG_2024-06-05T08_25_33.bin
-┣ FIP_RawIso_2024-06-05T08_25_33.bin
-┣ FIP_RawR_2024-06-05T08_25_33.bin
-┣ FIP_ROIsG-Iso_2024-06-05T08_25_33.csv
-┗ FIP_ROIsR_2024-06-05T08_25_33.csv
+┣📂 SoftwareEvents
+┃ ┣📜RepositoryStatus.json
+┃ ┗📜Region.json
+┣ green.csv
+┣ red.csv
+┣ iso.csv
+┣ green.bin
+┣ red.bin
+┣ iso.bin
+┣ roi_green_iso.csv
+┣ roi_red.csv
+┣ camera_green_iso_metadata.csv
+┗ camera_red_metadata.csv
 ```
-The file names are `FIP_[channel-name]_[datetime].csv`.  The `[channel-name]` token can be one of the following:
 
-* `DataG`: data for green channel
-* `DataR`: data for red channel
-* `DataIso`: data for isosbestic channel
+### CSV Files
 
-#### Photometry Readout CSV files
+The fiber photometry rig outputs CSV files containing metadata and data. All metadata and data files must contain headers. The order of the headers will not be enforced. Documented below for each type of CSV, the file and header names are specified.
 
-These files contain photometry readouts. Each row describes the average signal of pixels within each ROI for a single video frame. The values are computed online during data acquisition. The files have no headers. Each column is a timeseries, ordered as follows:
+#### Photometry CSV files
+
+Data files are named by the color of the channel and contain the photometry readouts.
+
+* `green.csv`: data for green channel
+* `red.csv`: data for red channel
+* `iso.csv`: data for isosbestic channel
+
+Each row describes the average signal of pixels within each ROI for a single video frame. The values are computed online during data acquisition. The headers include:
+
+* `CPUTime` (software timestamp, in milliseconds, total time of the day)
```

Comment:
CpuTime is NOT included in these files according to the current spec (and I don't think it should). If people want to recover them I included them in the `camera_*****_metadata.csv`.